src/Fieldtypes/Replicator.php

```diff
@@ -18,14 +18,20 @@ protected function configFieldItems(): array
                 'instructions' => __('statamic::fieldtypes.replicator.config.collapse'),
                 'type' => 'select',
                 'cast_booleans' => true,
-                'width' => 50,
+                'width' => 25,
                 'options' => [
                     'false' => __('statamic::fieldtypes.replicator.config.collapse.disabled'),
                     'true' => __('statamic::fieldtypes.replicator.config.collapse.enabled'),
                     'accordion' => __('statamic::fieldtypes.replicator.config.collapse.accordion'),
                 ],
                 'default' => false,
             ],
+            'max_sets' => [
+                'display' => __('Max Sets'),
+                'instructions' => __('statamic::fieldtypes.replicator.config.max_sets'),
+                'type' => 'integer',
+                'width' => 25,
```

Comment:
Remove the width, just let it default to full width. We'll tweak it later if need be.